src/main/java/com/cupid/jikting/common/util/RandomTeamNameGenerator.java

```diff
@@ -0,0 +1,35 @@
+package com.cupid.jikting.common.util;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class RandomTeamNameGenerator {
+
+    private static final String[] COLORS = new String[]{
+            "하양", "검정", "빨강", "주황", "노랑", "초록", "파랑", "보라", "분홍"
+    };
+    private static final String[] ANIMALS = new String[]{
+            "강아지", "고양이", "오리", "사자", "호랑이", "판다", "코끼리", "기린", "원숭이", "캥거루", "악어", "상어", "사슴", "토끼",
+            "거북이", "여우", "쿼카", "앵무새", "다람쥐", "너구리", "알파카", "치타", "펭귄", "햄스터", "미어캣", "수달", "표범"
+    };
+    private static final int RANDOM_NUMBER_RANGE = 10000;
+
+    public static String generate() {
+        return String.format("%s%s%04d",
+                COLORS[getSecureRandom().nextInt(COLORS.length)],
+                ANIMALS[getSecureRandom().nextInt(ANIMALS.length)],
+                getSecureRandom().nextInt(RANDOM_NUMBER_RANGE));
```

Comment:
p2. `StringBuilder`가 `String.format`에 비해 3배정도 빠르다고 합니다.
```suggestion
        StringBuilder sb = new StringBuilder();
        sb.append(COLORS[secureRandom.nextInt(COLORS.length)]);
        sb.append(ANIMALS[secureRandom.nextInt(ANIMALS.length)]);
        sb.append(String.format("%04d", secureRandom.nextInt(RANDOM_NUMBER_RANGE)));
        return sb.toString();
```